.github/workflows/ci.yml

```diff
@@ -218,3 +218,25 @@ jobs:
 
       - name: Run bindgen tests
         run: cargo test -p spacetimedb-cli
+
+  update:
+    name: Test spacetimedb-update flow
+    permissions: read-all
+    strategy:
+      matrix:
+        runner: [ spacetimedb-runner, windows-latest ]
```

Comment:
Can we run this on macos as well? I think at a minimum it should be tested on our major platforms which would be:
1. Windows x64
2. MacOS ARM
3. Linux x64
4. Linux ARM

Reply:
Good point. Let's see if it works!